__tests__/lib/build.js

```diff
@@ -1,32 +1,66 @@
-/* globals afterEach, beforeEach, describe, it, expect */
-const fs = require('fs')
-
+/* globals afterEach, beforeEach, describe, it, expect, jasmine */
 const build = require('../../lib/build')
-
 const util = require('../test-utils/util.js')
 
+const originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL
+
 describe('build', () => {
-  const buildFilePattern = 'built-build'
-  const cleanFn = util.makeCleanBuiltFilesFn(buildFilePattern)
   const mockDir = util.mockDir
 
-  beforeEach(cleanFn)
-  afterEach(cleanFn)
+  beforeEach(() => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000
+  })
+  afterEach(() => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout
+  })
+
+  describe('development', () => {
+    it('should transform js', (done) => {
+      const results = build({
+        config: {},
+        files: [[`${mockDir}/index.js`]]
+      })
 
-  it('should transform js and css', (done) => {
-    build({
-      config: {},
-      files: [
-        [`${mockDir}/mockComponent.js`, `${mockDir}/${buildFilePattern}.js`],
-        [`${mockDir}/mock.css`, `${mockDir}/${buildFilePattern}.css`]
-      ]
+      results[0]
+        .then((buf) => {
+          expect(buf.toString()).toMatchSnapshot()
```

Comment:
These snapshots produce a massive incomprehensible output.  Perhaps a better solution would be to make sure a keyword exists within the output.